src/store/chat.ts

```diff
@@ -1,5 +1,5 @@
+import type {  TextMessage } from "@lib/envelope/message";
 import { storage } from '@lib/services/store/store';
-import type { TextMsg } from '@lib/types/jam';
-import type { WSMsg } from '@lib/types/websocket';
 
-export const ChatStore = storage<WSMsg<TextMsg>[]>('CHAT_STORE', []);
+// TODO -- omitting `type` as not really needed here
```

Comment:
What type? `TextMessage` here? 
```ts
storage<TextMessage[]>('CHAT_STORE', [])
``` 
Don't we want to signal that this particular `storage` is used for TextMessages?
Or are you referring to something else?

Reply:
`TextMessage` is the same signature to `WSMsg<TextMsg>` but this PR makes do without the latter along with the `Envelope` type